server/pbenchinacan/load_keycloak.sh

```diff
@@ -33,14 +27,15 @@ REALM=${KEYCLOAK_REALM:-"pbench-server"}
 CLIENT=${KEYCLOAK_CLIENT:-"pbench-client"}
 
 PB_DEPLOY_FILES=${PB_DEPLOY_FILES:-"/var/tmp/pbench/pbench_server_deployment"}
+export CURL_CA_BUNDLE="${PWD}/server/pbenchinacan/etc/pki/tls/certs/pbench_CA.crt"
```

Comment:
I suspect that we want to leave the user (and higher-level scripts) the ability to control this if they want to.  So, this should follow the defaulting approach of the previous lines:
```sh
export CURL_CA_BUNDLE=${CURL_CA_BUNDLE:-"${PWD}/server/pbenchinacan/etc/pki/tls/certs/pbench_CA.crt"}
```